### ISSUE TITLE
[Suggestion]: 

### ISSUE THREAD

**Issue by sabeerbikba** [reactjs/react.dev#7074]
### Summary

Fix: the broken link in the [README.md](https://github.com/facebook/react/blob/main/packages/react-html/README.md) for the react-html package.

### Page

https://react.dev/reference/react-html

### Details

The [README.md](https://github.com/facebook/react/blob/main/packages/react-html/README.md) file for the `react-html` package contains a broken link that leads to a 404 Not Found page. It would be helpful to update or correct this link to ensure users can access the intended resource.

![IMG_20240721_053253](https://github.com/user-attachments/assets/e08c0403-821a-451c-a9ba-8a3507e84a0e)

**Comment by Jay-Karia** [comment]
I think you are in the wrong repo.
The broken link was mentioned in this repo: https://github.com/facebook/react
Try opening this issue there. 🙃

**Comment by sabeerbikba** [comment]
> I think you are in the wrong repo. The broken link was mentioned in this repo: https://github.com/facebook/react Try opening this issue there. 🙃

No, the React documentation is managed by this repository. The [facebook/react](https://github.com/facebook/react) repository is for managing the React library itself.

**Comment by Jay-Karia** [comment]
The [README](https://github.com/facebook/react/blob/main/packages/react-html/README.md) file which you mentioned belongs to [facebook/react]( https://github.com/facebook/react)

**Comment by Jay-Karia** [comment]
Are we closing ?
What's the matter ?
😕

**Comment by rickhanlonii** [comment]
The new package name is `react-markup`, though the link is still broken:

https://github.com/facebook/react/blob/cea13feb3449317d8570ba17603747497037d264/packages/react-markup/README.md#L32

cc @eps1lon, this should probably say there are no docs until there are actually docs.
